jsBasics/lesson22/src/fixtures/login.fixture.js

```diff
@@ -0,0 +1,29 @@
+import { test as base } from '@playwright/test';
+import { LoginPage, GaragePage } from '../pageObjects';
+import { expect } from '@playwright/test';
+
+export const test = base.extend({
+  /**
+   * @type {import('../pageObjects').GaragePage}
+   */
+
+  garagePage: async ({ page, context }, use) => {
+    const garagePage = new GaragePage(page, context);
+    await garagePage.navigateToPage('/panel/garage');
+    await garagePage.clickAddCarButton();
+    // await expect(garagePage.locators.addBtnPopup).toBeVisible()
+    //await garagePage.fillInCarForm('BMW', 'X5', '555');
+    await garagePage.locators.brandDrpdwn.selectOption('BMW');
+    await garagePage.locators.modelDrpdwn.selectOption('X5');
+    await garagePage.locators.addCarMlg.fill('555');
+    await garagePage.locators.addBtnPopup.click();
+    await use(garagePage);
```

Comment:
here after the use() you can bypass deleteAllCars, so that it is automatically used in afterEach.
Just be careful, it'd be disruptive for cases when this tests run in parallel, breaking neighbouring test. I'd suggest on long run capture ID of the car and delete only one with particular ID, via API for example